docs/getting-started/continuous_integration.md

```diff
@@ -59,19 +59,21 @@ jobs:
    runs-on: ubuntu-latest
    steps:
    - name: Build Fuzzers
+     id: build
      uses: google/oss-fuzz/infra/cifuzz/actions/build_fuzzers@master
      with:
        oss-fuzz-project-name: 'example'
        dry-run: false
    - name: Run Fuzzers
+     id: run
      uses: google/oss-fuzz/infra/cifuzz/actions/run_fuzzers@master
      with:
        oss-fuzz-project-name: 'example'
        fuzz-seconds: 600
        dry-run: false
    - name: Upload Crash
      uses: actions/upload-artifact@v1
-     if: failure()
+     if: failure() && steps.build.outcome == 'success' && steps.run.outcome != 'success'
```

Comment:
Can you change to `failure() && steps.build.outcome == 'success'`
I spent some time trying to figure out why `steps.run.outcome == 'failure' && steps.build.outcome == 'success'`
but couldn't. If you know why I'd appreciate it a lot.

Reply:
I agree that part is redundant at this point. I decided to keep it because it might come in handy should another step that could fail be added (which is unlikely to happen though). I'll change that. Plus with that part gone, "run" will no longer be referred to so it should be safe to just drop "id: run".